src/view/screens/Home.tsx

```diff
@@ -80,14 +82,18 @@ function HomeScreenReady({
   const pagerRef = React.useRef<PagerRef>(null)
   const lastPagerReportedIndexRef = React.useRef(selectedIndex)
   React.useLayoutEffect(() => {
+    let initialIndex = selectedIndex
     // Since the pager is not a controlled component, adjust it imperatively
     // if the selected index gets out of sync with what it last reported.
     // This is supposed to only happen on the web when you use the right nav.
-    if (selectedIndex !== lastPagerReportedIndexRef.current) {
+    if (initialIndex !== selectedIndex) {
```

Comment:
Is this set of changes still needed? I think maybe not. If I'm right, let's fully undo this file.

Reply:
Oh yea, should be reverted 👍